Gems/Atom/Asset/ImageProcessingAtom/Code/Source/ImageLoader/QtImageLoader.cpp

```diff
@@ -57,6 +73,170 @@ namespace ImageProcessingAtom
             return pImage;
         }
 
+        // https://forum.qt.io/topic/74712/qimage-from-tga-with-alpha/11
+        // https://forum.qt.io/topic/101971/qimage-and-tga-support-in-c/5
+        static bool loadTga(const AZStd::string& filename, QImage& image)
+        {
+            if (!image.load(filename.c_str()))
+            {
+                // open the file
+                FILE* pf = nullptr;
+                azfopen(&pf, filename.c_str(), "rb");
+
+                if (!pf)
+                {
+                    return false;
+                }
+
+                // some variables
+                AZStd::vector<uint8_t>* vui8Pixels = nullptr;
```

Comment:
Suggestion: 
you can use AZStd::vector<uint8_t> vui8Pixels; here instead of object. This would simplify the code (multiple places which to release the pointer.